### PR TITLE
:sparkles: [#1637] added 2fa

### PR DESCRIPTION
**Pull request by bart-maykin**
Fixes [#1637](https://github.com/open-zaak/open-zaak/issues/1637)

Added 2fa

### CODE REVIEW

File: src/openzaak/templates/maykin_2fa/base.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+{% comment %} SPDX-License-Identifier: EUPL-1.2 {% endcomment %}
+{% comment %} Copyright (C) 2019 - 2020 Dimpact {% endcomment %}
+
+{# Django 3.2 #}
+{% block breadcrumbs %}{% endblock %}
+
+{# Do not show any version information #}
+{% block footer %}
+    <div id="footer"></div>
+{% endblock %}
```
Comment (stevenbal) [inline src/openzaak/templates/maykin_2fa/base.html:11]: Same comment as for ON, this file can be removed

File: src/openzaak/templates/maykin_2fa/login.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+{% block extra_recovery_options %}
+<li>
+    {% trans 'Contact support to start the account recovery process' %}
+</li>
+{% endblock extra_recovery_options %}
+
+{# Do not show any version information #}
+{% block footer %}
+    <div id="footer"></div>
+{% endblock %}
```
Comment (stevenbal) [inline src/openzaak/templates/maykin_2fa/login.html:25]: Same comment as for ON, this block can be removed
```suggestion
```

File: src/openzaak/templates/maykin_2fa/base.html (hunk 1 above)
Comment (stevenbal) [reply src/openzaak/templates/maykin_2fa/base.html]: Sorry, I was wrong about removing the `base.html` file, it turns out Open Zaak does use this `footer` block https://github.com/open-zaak/open-zaak/blob/e701f8155735949033fc53f0c78ac4ce9fcc97ca/src/openzaak/templates/admin/base_site.html#L43 (while Open Notificaties does not). Could you re-add it here?

**Review by stevenbal — CHANGES_REQUESTED** [review]
Just the last comment to re-add `base.html`, then I'll approve

**Review by stevenbal — APPROVED** [review]

**Comment by annashamray** [comment]
What about backward compatibility? Now we will force all our clients to set up 2fa for the admin without any notice. 
@bart-maykin please make it possible to disable it for now. 
We will announce this feature but not make it required this time

**Pull request merged**